Shopify/hydrogen
# Fix changing currency on cart mutation

### WHY are these changes introduced?

Still unable to determine the cause of this issue: When country is supplied to `@inContext` for cart mutations, it will update the buyer identity country of the cart.

The buyer identity country should not change on cart mutation.

Reproducible on hydrogen.shop:

1. Go to hydrogen.shop
2. Add an item to cart
3. Change country to Canada (currency display in CAD)
4. Update quantity of item in cart

See currency of cart changed back to USD

### WHAT is this pull request doing?

* Updates demo store so that any cart mutation form is using the localized path
* Update `hydrogen setup` command for subfolder i18n strategy so that any `CartForm` component is using the localized path

### HOW to test your changes?

In the `template/skeleton` folder, run `npx shopify hydrogen setup`, select `Subfolders` and select `No` for `Scaffold all standard route files`. You should see:

* `root.tsx` 
  * added const `DEFAULT_LOCALE`
  * added export `usePrefixPathWithLocale`
  * added `selectedLocale` to loader return
* `Cart.tsx`
  * imported `usePrefixPathWithLocale`
  * updated `<CartForm>` property `route` with the result from `usePrefixPathWithLocale("/cart")`
*  `AddToCartButton.tsx`
   * imported `usePrefixPathWithLocale`
   * updated `<CartForm>` property `route` with the result from `usePrefixPathWithLocale("/cart")`

#### Post-merge steps



#### Checklist
